### PR TITLE
Adding Scoring and (correct/wrong Banner)

### PR DESCRIPTION
**Pull request by husseinfaqih**

### CODE REVIEW

File: desktop.ini [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[.ShellClassInfo]
```
Comment (mnvacym) [inline desktop.ini:1]: Why do you need this file?

File: src/utils/Banner.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+///>>> Creating a Correct Answer Banner
+import { USER_INTERFACE_ID } from '../constants.js';
+const userInterface = document.getElementById(USER_INTERFACE_ID);
+
+export function correctBanner() {
+  const correctAnswerBanner = document.createElement('p');
+  correctAnswerBanner.textContent = 'Correct Answer';
+  correctAnswerBanner.style.display = 'block';
```
Comment (mnvacym) [inline src/utils/Banner.js:8]: Can you add this style in css?

File: src/pages/questionPage.js [hunk 3]
```diff
@@ -6,8 +6,12 @@ import {
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
+// import { appendLinks } from '../utils/appendLinksUtil.js';
```
Comment (mnvacym) [inline src/pages/questionPage.js:9]: Is this function not needed?

File: src/utils/Banner.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  correctAnswerBanner.className = 'banner-correct'; // TO STYLE IT LATER
+  // document.body.appendChild(correctAnswerBanner); // AT THE TOP OF THE PAGE
```
Comment (mnvacym) [inline src/utils/Banner.js:10]: If you don't need this line please delete it

File: src/utils/Banner.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  userInterface.appendChild(correctAnswerBanner);
+} // AT THE BOTTOM  OF THE PAGE
+
+///>>> Creating a Correct Answer Banner
+export function wrongBanner() {
+  const wrongAnswerBanner = document.createElement('p');
+  wrongAnswerBanner.textContent = 'Wrong Answer';
+  wrongAnswerBanner.style.display = 'block';
+  wrongAnswerBanner.className = 'banner-wrong'; // TO STYLE IT LATER
+  // document.body.appendChild(correctAnswerBanner); // AT THE TOP OF THE PAGE
```
Comment (mnvacym) [inline src/utils/Banner.js:20]: If you don't need this line please delete it

File: desktop.ini (hunk 1 above)
Comment (husseinfaqih) [reply desktop.ini:1]: I didn't modify the folder desktop.ini so I am not sure how I ended up doing this!

File: src/utils/Banner.js (hunk 2 above)
Comment (husseinfaqih) [reply src/utils/Banner.js:8]: I noticed that this line is not needed anymore, so I am deleting it

File: src/pages/questionPage.js (hunk 3 above)
Comment (husseinfaqih) [reply src/pages/questionPage.js:9]: I was getting a notification :
'appendLinks' is declared but its value is never read.'
I believe whoever added it didn't use it at the end! So I commented it out. to double check

**Comment by husseinfaqih** [comment]
@mnvacym I updated the files based on your feedback!

**Review by mnvacym — APPROVED** [review]

**Pull request merged**